sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -155,6 +178,34 @@ def discovery_url(self) -> Optional[str]:
         """
         return self._discovery_url
 
+    # Exists to appease tox's mypy rules.
+    @property
+    def _hub_id(self) -> Optional[str]:
+        """The UID of the hub parent of the project. This is an internal property
+        that's surfaced by the Project sub-class, but exists here for backwards-compatibility
+        reasons.
+
+        :return: Resource ID of the parent hub.
+        :rtype: str
+        """
+        return self.__hub_id
+
+    # Exists to appease tox's mypy rules.
+    @_hub_id.setter
+    def _hub_id(self, value: str):
+        """Set the hub of the project. This is an internal property
+        that's surfaced by the Project sub-class, but exists here for backwards-compatibility
+        reasons.
+
+
+        :param value: The hub id to assign to the project.
+            Note: cannot be reassigned after creation.
+        :type value: str
+        """
+        if not value:
+            return
+        self.__hub_id = value
+
```

Comment:
What's the intent with this property?

As written, `_hub_id` (the property) acts just like a object attribute that can't be set to the empty string (or any falsy value).

The docstring makes it seem like this is meant to be read-only, but that isn't what the code is doing.

Reply:
Answered in Hanchi's call, TL:DR it's making mypy shut up.